recipes/xmlsec/all/test_package/conanfile.py

```diff
@@ -1,19 +1,25 @@
 from conan import ConanFile
 from conan.tools.build import can_run
-from conan.tools.cmake import CMake, cmake_layout
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
 import os
 
 
 class TestPackageConan(ConanFile):
     settings = "os", "arch", "compiler", "build_type"
-    generators = "CMakeDeps", "CMakeToolchain", "VirtualRunEnv"
+    generators = "CMakeDeps", "VirtualRunEnv"
     test_type = "explicit"
 
     def layout(self):
         cmake_layout(self)
 
     def requirements(self):
         self.requires(self.tested_reference_str)
+        self.requires("libxml2/2.10.4")
```

Comment:
IT would have been nice to have https://github.com/conan-io/conan-center-index/pull/17062/commits/dc5da99d483ccc2613487892ad094621d34272aa here since it's out of the norm